### PR TITLE
Port Gaeguli to GStreamer 1.16

### PR DESCRIPTION
**Pull request by xhaakon**

### CODE REVIEW

**Comment by xhaakon** [comment]
@justinjoy This is merely FYI now. Still needs some work in the terms of the test suite.

**Comment by justinjoy** [comment]
> @justinjoy This is merely FYI now. Still needs some work in the terms of the test suite.

No problem. Just tell me when you think these are ready. :)

Just a quick note. As you can see, CI node at Azure was dropped. 
All of repository is transiting to github action so some of unit tests could be failed due to wrong CI environment settings.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
+*[gæguli]* is a SRT streamer designed for edge devices that require strong security and ultra-low latency.
 
 ## Overview
-*Gaeguli* implements the supporting library to handling SRT streaming. It is designed with two parts
-*   pipeline: Reads streaming for a video source and injects it into a fifo
-*   fifo-transmitter: Reads fifo and send streaming to a specific URI using SRT protocol
-
-So basically generation and transmission are decoupled.
+*Gaeguli* implements the supporting library for handling SRT streaming. It provides 'pipeline' component
+that deals with receiving frames from a video source and streaming to a specific URI using SRT protocol.
```
Comment (justinjoy) [inline README.md:8]: 👍

**Review by justinjoy — APPROVED** [review]
